src/Toggle/MysqlGetAllTogglesGateway.php

```diff
@@ -13,15 +13,107 @@
 
 abstract class MysqlGetAllTogglesGateway
 {
+
     /**
-     * @param array $data
+     * @param $row
+     * @return string
+     */
+    protected function getDefaultForScreenshotUrl( $row )
+    {
+        $screenshotUrl = isset( $row[ 'screenshot_urls' ] ) ? $row[ 'screenshot_urls' ] : null;
```

Comment:
In all these getDefault methods it's quite unnecessary to assign the value to a local variable, you could just return with the result right away.